### ISSUE TITLE
Remove javadoc for overridden methods

### ISSUE THREAD

**Issue by gregw** [eclipse/jetty.project#4581]
There are many methods in the source that have javadoc like:
```java
    /*
     * @see javax.servlet.SomeClass#someMethod()
     */
    @Override
    public Object someMethod()
    {
        ...
    }
```

We should remove all such empty javadoc as the `@Override` provides all the information needed.

**Comment by gregw** [comment]
@joakime @olamy @lachlan-roberts  Do you have any awk/sed/scripting magic that can do this?

**Comment by joakime** [comment]
Not offhand, but I wonder if checkstyle (or some other tool) can detect javadoc that has no content like that?

**Comment by lachlan-roberts** [comment]
I tried running a few different regex replaces through Intellij, I'll put a PR up and see how it looks.

**Comment by lachlan-roberts** [comment]
Fixed in jetty-10 with PR #4583.
